tests/test_hedgehog_smoke.py

```diff
@@ -273,6 +273,14 @@ def test_swss_smoke(setup):
 
     pytest_assert(is_process_running(duthost, "orchagent"), "There is no running orchagent process.")
 
+def test_pmon_smoke(setup):
+    duthost = setup['duthost']
+    config = setup['config']
+    container = "pmon"
+
+    check_container_sanity_helper(config, container)
+
+    check_container_restarts_helper(duthost, container)
```

Comment:
there is no process which can be verified ?

Reply:
I don't see any process with name related to pmon